assets/css/_sass/website/_docs-page.scss

```diff
@@ -54,9 +58,8 @@
     font-size: inherit;
     color: $grey900;
   }
-  article {
-    padding: 0 28px;
-    margin: 0;
+  .content-wrapper {
```

Comment:
Is this `.content-wrapper` inside of a `.content-wrapper`? Was that intentional?

Reply:
ah -- nope, the previous structure was:

.docs-content {
  article {}
}

this was an error on my part.  I should have changed the second, nested .content-wrapper to .docs-content.

Will change